api/server.go

```diff
@@ -70,6 +74,8 @@ type (
 	}
 )
 
+var startTime = time.Now()
```

Comment:
Small nit but this could be on the `server` since it's only used by the server. It would also make it easier to more accurately check the start time in a test since every server would store its own start time.